packages/packages/api/src/medical/client/metriport.ts

```diff
@@ -354,16 +354,18 @@ export class MetriportMedicalApi {
   }
 
   /**
-   * Returns a URL that can be used to download the document.
+   * Fetches the document from S3 and sends a presigned URL
    *
-   * @param fileName The file name of the document in s3.
-   * @return presigned url that expires in 20 seconds
+   * @param req.query.fileName The file name of the document in s3.
+   * @param req.query.conversionType The doc type to convert to.
```

Comment:
We should list the valid values, ideally also making a union type on `conversionType` (`pdf` | `xml`).